notebooks/nessie-iceberg-demo-nba.ipynb

```diff
@@ -0,0 +1,629 @@
+    "          Season STRING, Age STRING, Team STRING, ORB STRING, TRB STRING, AST STRING, STL STRING, BLK STRING, \n",
+    "          TOV STRING, PF STRING, PTS STRING, Player STRING) \n",
+    "          USING iceberg\"\"\")\n",
+    "spark.sql('CREATE OR REPLACE TEMPORARY VIEW allstar_table USING csv OPTIONS (path \"../datasets/nba/allstar_games_stats.csv\", header true)')\n",
+    "spark.sql('INSERT INTO dev_catalog.nba.allstar_games_stats SELECT * FROM allstar_table').toPandas()\n",
+    "\n",
+    "# notice how we view the data on the etl branch via @etl\n",
+    "spark.sql(\"select count(*) from dev_catalog.nba.`allstar_games_stats@etl`\").toPandas()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can verify that the new table isn't on the `main` branch but is present on the etl branch"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "spark.sql(\"USE REFERENCE main IN dev_catalog\").toPandas()\n",
+    "spark.sql(\"SHOW TABLES IN dev_catalog\").toPandas()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "spark.sql(\"USE REFERENCE etl IN dev_catalog\").toPandas()\n",
+    "spark.sql(\"SHOW TABLES IN dev_catalog\").toPandas()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that we are happy with the data we can again merge it into `main`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "jupyter": {
+     "outputs_hidden": false
+    },
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "spark.sql(\"MERGE BRANCH etl IN dev_catalog\").toPandas()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now lets verify that the changes exist on the `main` branch and that the `main` and `etl` branches have the same `hash`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "jupyter": {
+     "outputs_hidden": false
+    },
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "spark.sql(\"USE REFERENCE main IN dev_catalog\").toPandas()\n",
+    "spark.sql(\"SHOW TABLES IN dev_catalog\").toPandas()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "spark.sql(\"LIST REFERENCES IN dev_catalog\").toPandas()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "table_count = spark.sql(\"select count(*) from dev_catalog.nba.allstar_games_stats\").toPandas().values[0][0]\n",
+    "csv_count = spark.sql(\"select count(*) from allstar_table\").toPandas().values[0][0]\n",
+    "assert table_count == csv_count"
```

Comment:
Same here: Maybe add some `print` of these numbers here, so people have something to look at.

Reply:
good idea, done